test/runnable/foreach4.d

```diff
@@ -841,7 +841,7 @@ void test26()
     assert(a.array[0] == 73);
     assert(a.array[1] == 82);
 
-    foreach (uint u; &a.reverse)
+    foreach (uint u; &a.forward)
```

Comment:
Changed this to `forward` instead of `reverse`, because it seems that the idea was to test the `uint` index type, but the result of the operation is ignored.  Besides, Phobos `reverse` does not return the array, so it can't be substituted here.